Cpp/odin/app.cpp

```diff
@@ -53,5 +56,17 @@ fostlib::jwt::mint odin::app::mint_user_jwt(
             + fostlib::coerce<fostlib::string>(app[app_id]);
     fostlib::jwt::mint jwt{fostlib::sha256, jwt_secret, std::move(payload)};
     jwt.subject(fostlib::coerce<fostlib::string>(user[subject]));
+    jwt.claim("iss", app[app_id]);
     return jwt;
 }
+
+void odin::app::save_app_user(
+    fostlib::pg::connection &cnx, f5::u8view reference,
+    const fostlib::json &user, const f5::u8view app_id
+) {
+    fg::json app_user_values;
+    fostlib::insert(app_user_values, "reference", reference);
+    fostlib::insert(app_user_values, "identity_id", user["identity"]["id"]);
```

Comment:
Probably better to have the `identity_id` passed in like we do for the `app_id`, then we're not reliant on particular JSON structures.